spire/templates/apps/augury.yml

```diff
@@ -1170,6 +1180,13 @@ Resources:
         // ECS runTask parameters
         const runParams = (name) => {
           return {
+            capacityProviderStrategy: [
```

Comment:
So is the idea that... when we move Augury over to arm, we'll need the Fargates to also launch using that capacity provider?

Reply:
I think we'd actually want to set [this runtime config](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-runtimeplatform.html) on the task def. I forgot how Fargate worked.